sections/related_works.tex

```diff
@@ -0,0 +1,37 @@
+\section{Related Work}\label{sec:related-works} \vejleder[inline]{add some more}
+In this section, we investigate existing literature concerning \gls{lda} and similar topic models, as well as relevant work related to random walk.
+
+\citet{lda} describes a generative statistical model, which is heavily used within topic modeling. 
+The intuition behind the \gls{lda} topic model is that a document has an underlying distribution of topics, and that the words of a document are based on these topics.
+This model is used widely within the field of topic modeling and is somewhat considered a standard, see section \autoref{sec:lda} for more detail.
+
+There also exists a large amount of extensions to \gls{lda}.
+\citet{blei2012topicmodels} describes some of the most significant extensions that are made from relaxing the assumptions of \gls{lda}.
```

Comment:
perhaps we should also mention all of these assumptions in our lda section to clarify its weaknesses.